--- a/usr/share/texlive/texmf-dist/tex/generic/pgf/systemlayer/pgfsys-dvisvgm.def	2024-02-22 13:30:26.167777811 -0600
+++ b/usr/share/texlive/texmf-dist/tex/generic/pgf/systemlayer/pgfsys-dvisvgm.def	2024-02-22 13:36:29.820956330 -0600
@@ -127,7 +127,15 @@
   \dp#1=0pt%
   \leavevmode%
   \pgf@xa=\pgf@trimleft@final\relax  \ifdim\pgf@xa=0pt \else\kern\pgf@xa\fi%
-  \raise-\pgf@ya\hbox{\ifpgf@sys@svg@inpicture\else\special{dvisvgm:bbox \pgf@sys@tonumber\pgf@picmaxx\space\pgf@sys@tonumber\pgf@picmaxy}\fi\box#1}%
+  \raise-\pgf@ya\hbox{%
+    \ifpgf@sys@svg@inpicture
+      \box#1%
+    \else
+      \special{dvisvgm:bbox \pgf@sys@tonumber\pgf@picmaxx\space\pgf@sys@tonumber\pgf@picmaxy}%
+      \special{dvisvgm:bbox lock}%
+      \box#1%
+      \special{dvisvgm:bbox unlock}%
+    \fi}%
   \pgf@xa=\pgf@trimright@final\relax \ifdim\pgf@xa=0pt \else\kern\pgf@xa\fi%
 }

